### ISSUE TITLE
Persistent local storage

### ISSUE THREAD

**Issue by jakolehm** [kontena/pharos-cluster#104]
We should provide some sane way to consume host local storage.

See:
- https://kubernetes.io/docs/concepts/storage/storage-classes/#local
- https://github.com/kubernetes-incubator/external-storage/tree/master/local-volume/provisioner

**Comment by wtip** [comment]
Are there still plans to add this? Since the feature gates are no longer needed on K8s 1.10 I guess you can just follow the local-volume-provisioner docs to set things up.

**Comment by jakolehm** [comment]
Yes, with Kubernetes 1.10 (Pharos 1.1) you can just follow the local persistent volume docs and optionally deploy local-volume-provisioner.

That said I still think this might be quite handy add-on to have... but it's too late for 1.2.x release cycle.
